src/sync/installation.ts

```diff
@@ -214,7 +222,7 @@ export const isRetryableWithSmallerRequest = (err): boolean => {
 // Checks if parsed error type is NOT_FOUND / status is 404 which come from 2 different sources
 // - GraphqlError: https://github.com/octokit/graphql.js/tree/master#errors
 // - RequestError: https://github.com/octokit/request.js/blob/5cef43ea4008728139686b6e542a62df28bb112a/src/fetch-wrapper.ts#L77
-export const handleNotFoundErrors = (
+export const isNotFoundError = (
```

Comment:
Renamed this function, as it's not handling anything, but rather checking if it's a not found error.